cdap-api-common/src/main/java/io/cdap/cdap/api/data/schema/SchemaHash.java

```diff
@@ -41,14 +41,19 @@
   private String hashStr;
 
   public SchemaHash(Schema schema) {
-    hash = computeHash(schema);
+    this(schema, true);
+  }
+
+  public SchemaHash(Schema schema, boolean includeRecordName) {
+    hash = computeHash(schema, includeRecordName);
   }
 
   /**
    * Creates a SchemaHash by reading the hash value from the given {@link ByteBuffer}.
    * The position of the {@link ByteBuffer} will be moved to the byte after the hash
    * value.
    */
+  @Deprecated
```

Comment:
Why this is deprecated?

Reply:
was a intermediate thing i forgot to remove, will remove